llvm/tools/llvm-cov/CodeCoverage.cpp

```diff
@@ -1023,6 +1023,11 @@ int CodeCoverageTool::doShow(int argc, const char **argv,
   cl::alias ShowOutputDirectoryA("o", cl::desc("Alias for --output-dir"),
                                  cl::aliasopt(ShowOutputDirectory));
 
+  cl::opt<bool> BinaryCounters(
+      "binary-counters", cl::Optional,
+      cl::desc("Show 1/0 instead of actual counter values."),
```

Comment:
I think it might help coverage noobs a little to make it clear that 1 == covered, 0 == covered.

> Show when source is covered (1) or uncovered (0) instead of showing actual counter values.

Reply:
Done, with rewording `lines/branches` since counters belong line and branches.